### PR TITLE
added about offer block to offerDetails page

### PR DESCRIPTION
**Pull request by Tolik170**
image
image

### CODE REVIEW

File: src/components/show-more-collapse/ShowMoreCollapse.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <Typography sx={styles.description(collapsedHeight)}>
+      {expanded ? description : description.slice(0, collapsedTextLength)}
+    </Typography>
+  )
+
+  return (
+    <Box>
+      <Typography sx={styles.title}>{title}</Typography>
+
+      <Collapse collapsedSize={collapsedSize} in={expanded} {...props}>
```
Comment (mxrcury) [inline src/components/show-more-collapse/ShowMoreCollapse.tsx:44]: if you want to add here props that Collapse component from MUI receives, so it i think it needs to add extending of CollapseProps interface from MUI to ShowMoreCollapseProps

File: src/constants/translations/en/common.json [hunk 2]
```diff
@@ -10,13 +10,16 @@
...
+  "showMore": "Show more",
+  "showLess": "Show less",
   "confirmButton": "OK",
   "uah": "UAH",
   "hour": "HOUR",
   "hourSlash": "/ HOUR",
   "beginner": "Beginner",
   "offers": "offers",
   "popularCategories": "Popular Categories",
+  "aboutOffer": "About Offer",
```
Comment (mxrcury) [inline src/constants/translations/en/common.json:22]: isn't it needed to add aboutOffer also to ua translations

File: src/components/show-more-collapse/ShowMoreCollapse.tsx (hunk 1 above)
Comment (Tolik170) [reply src/components/show-more-collapse/ShowMoreCollapse.tsx:44]: you are right, fixed

File: src/constants/translations/en/common.json (hunk 2 above)
Comment (Tolik170) [reply src/constants/translations/en/common.json:22]: Didn`t see it, fixed

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**